### PR TITLE
Update readme with information about C++ for OpenCL.

### PR DESCRIPTION
**Pull request by AnastasiaStulova**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -15,7 +15,8 @@ Dependencies>>.
 
 This repository contains the source and tool chain used to generate the
 formal OpenCL API, OpenCL Extensions, OpenCL SPIR-V Environment, and
-OpenCL C++ Specifications, as well as the OpenCL 2.2 Reference Pages.
+OpenCL C++ Specifications, as well as the OpenCL 2.2 Reference Pages
+and documentation for C++ for OpenCL kernel language.
```
Comment (svenvh) [inline README.adoc:19]:
```suggestion
and documentation for the C++ for OpenCL kernel language.
```

**Review by svenvh — APPROVED** [review]

**Review by bashbaug — APPROVED** [review]
You didn't break this, but it's odd that we don't mention the OpenCL C specs in this README also.

I'll file an issue to fix this.

**Comment by AnastasiaStulova** [comment]
> You didn't break this, but it's odd that we don't mention the OpenCL C specs in this README also.
> 
> I'll file an issue to fix this.

This is strange indeed. I haven't noticed that. I can't fix this in another PR...

**Comment by AnastasiaStulova** [comment]
> > You didn't break this, but it's odd that we don't mention the OpenCL C specs in this README also.
> > I'll file an issue to fix this.
> 
> This is strange indeed. I haven't noticed that. I can't fix this in another PR...

Here it is https://github.com/KhronosGroup/OpenCL-Docs/pull/589. Well spotted btw!

**Pull request merged**